cmd/postgres_exporter/server.go

```diff
@@ -128,6 +131,17 @@ func (s *Server) Scrape(ch chan<- prometheus.Metric, disableSettingsMetrics bool
 		err = fmt.Errorf("queryNamespaceMappings returned %d errors", len(errMap))
 	}
 
+	{
```

Comment:
This is in a block to not reset the err var that gets set above. I don't want to return early because I don't think the errors are fatal, just informational.